UnitTests/MappingResolution/EntityIdentityResolverTest.cs

```diff
@@ -109,37 +109,40 @@ public void ResolvePotentialEntity_EntityRefMember_OptimizesReferenceViaResolver
     {
       var fakeOptimizedIdentity = ExpressionHelper.CreateExpression();
       _resolverMock
-          .Expect (mock => mock.TryResolveOptimizedIdentity (_entityRefMemberExpression))
-          .Return (fakeOptimizedIdentity);
+         .Setup (mock => mock.TryResolveOptimizedIdentity (_entityRefMemberExpression))
+         .Returns (fakeOptimizedIdentity)
+         .Verifiable ();
       
       var result = _entityIdentityResolver.ResolvePotentialEntity (_entityRefMemberExpression);
 
-      _resolverMock.VerifyAllExpectations ();
+      _resolverMock.Verify ();
       Assert.That (result, Is.SameAs (fakeOptimizedIdentity));
     }
 
     [Test]
     public void ResolvePotentialEntity_EntityRefMember_NonOptimizable_ResolvesToJoinAndIdentity ()
     {
       _resolverMock
-        .Expect (mock => mock.TryResolveOptimizedIdentity (_entityRefMemberExpression))
-        .Return (null);
+         .Setup (mock => mock.TryResolveOptimizedIdentity (_entityRefMemberExpression))
+         .Returns ((Delegate) null) //REVIEW correct?
```

Comment:
Since the method has `Expression` as the return type, I'd cast null to `Expression` too